libs/smiles/environment.yml

```diff
@@ -0,0 +1,17 @@
+name: smiles
+channels:
+  - conda-forge
+dependencies:
+  - python=3.11
+  - requests=2.28.2
+  - pandas=1.5.3
+  - numpy=1.24.2
+  - tqdm=4.64.1
+  - rdkit=2022.9.4
```

Comment:
Oh, bizarre, no idea why I didn't find it. Sounds good to me!

Reply:
I hope it's ok to leave it as is (conda) mostly because I've run out of time :D Feel free to flag this and I can create an issue, if this is a concern.